src/assets/Doneth.sol.txt

```diff
@@ -192,13 +219,14 @@ contract Doneth {
         Withdraw(msg.sender, amount, totalWithdrawn);
     }
 
+
     // Withdraw from shared expense allocation. Total withdrawable is calculated as 
     // sharedExpense - sharedExpenseWithdrawn.
-    function withdrawSharedExpense(uint256 amount) public onlyAdmin() onlyExisting(msg.sender) {
+    function withdrawSharedExpense(uint256 amount, address to) public onlyAdmin() onlyExisting(msg.sender) {
```

Comment:
Are we sure we want to have a "to" address? That means that admins can siphon funds to different addresses that aren't the one they used to be a Member in the contract?

Reply:
i think that'd be a cool feature but happy to decide as a group. the code is almost the exact same but it means you don't have to do two transactions if you want to move it from your account address to a spending wallet or somewhere else. think that feature led me to imagine the expense account as a normal member who just didn't have a withdrawn amount or limit. @TrevorJTClarke you have any opinion about this?

Reply:
oops i'd been answering this question as if it were the normal withdraw (which i think could also benefit from a to-address but that's another conversation). but i think expense withdraw really does need a to-address. this seems like a much more common scenario, like you'd pay for hosting or office supplies by transferring directly to the merchant. if it was something you had paid yourself (like hotel/flight situation) then the to-address would just be the address you'd like to accept it with. think it shows more accountability as well as being better ease of use. what do you all think?